### ISSUE TITLE
[Tuning] panther_gsuite_helpers - Missing keyword argument for deep_get default value

### ISSUE THREAD

**Issue by zestysoft** [panther-labs/panther-analysis#1879]
## Rule ID

panther_gsuite_helpers (Global Helper)

## Description

The `gsuite_activityevent_alert_context` function in `global_helpers/panther_gsuite_helpers.py` has incorrect `deep_get` calls that pass the empty string `""` as a positional argument instead of using the `default=` keyword argument.

```python
def gsuite_activityevent_alert_context(event):
    return {
        "actor": event.deep_get("actor", "email", ""),           # BUG
        "applicationName": event.deep_get("id", "applicationName", ""),  # BUG
        "name": event.get("name", ""),
        "type": event.get("type", ""),
        "parameters": event.get("parameters", {}),
    }
```

The `deep_get` function signature is `def deep_get(dictionary: dict, *keys, default=None)`, meaning `default` is a keyword-only argument. The calls `event.deep_get("actor", "email", "")` and `event.deep_get("id", "applicationName", "")` pass `""` as a positional argument, which gets interpreted as a **third path key** rather than the default value.

This causes the function to look for:
- `actor -> email -> ""` (empty string as a key) instead of `actor -> email` with default `""`
- `id -> applicationName -> ""` instead of `id -> applicationName` with default `""`

The result is that these calls return `None` instead of `""` when the path doesn't exist, which is inconsistent with the intended behavior and with other `deep_get` usages in the codebase that correctly use `default=`.

**Suggested Fix:**

```python
def gsuite_activityevent_alert_context(event):
    return {
        "actor": event.deep_get("actor", "email", default=""),
        "applicationName": event.deep_get("id", "applicationName", default=""),
        "name": event.get("name", ""),
        "type": event.get("type", ""),
        "parameters": event.get("parameters", {}),
    }
```
